ovos_ocp_youtube_plugin/__init__.py

```
@@ -236,6 +236,12 @@ def supported_seis(self):
     def extract_stream(self, uri, video=True):
         """ return the real uri that can be played by OCP """
         uri = uri.replace("youtube.channel.live//", "")
+        if uri.endswith("/live"): # no parsing needed in theory
+            try:
+                return super().extract_stream(url)
```

Ruff (0.8.2)

Undefined name `url`

(F821)